target/linux/ipq40xx/image/generic.mk

```diff
@@ -190,6 +190,20 @@ define Device/aruba_ap-365
 endef
 TARGET_DEVICES += aruba_ap-365
 
+define Device/asus_map-ac1300
+	$(call Device/FitImageLzma)
+	DEVICE_VENDOR := ASUS
+	DEVICE_MODEL := Lyra Mini (MAP-AC1300)
+	SOC := qcom-ipq4018
+	DEVICE_PACKAGES := -kmod-ath10k-ct kmod-ath10k-ct-smallbuffers
```

Comment:
256MB ram, so why small buffers?

Reply:
Another point here: the officially committed version should probably have the Candela-Tech (`-ct`) drivers, but if you want 802.11s (mesh) you need to self-compile with the non `-ct` version.

Reply:
Please drop the wlan driver override from packages